nvtabular/ops/categorify.py

```diff
@@ -520,26 +524,40 @@ def _make_name(*args, sep="_"):
     return sep.join(args)
 
 
+@dataclass
+class _FitOptions:
```

Comment:
Should we add some docs here?

Reply:
I've added some brief descriptions of each parameters in the last commit .